src/main/java/muller/storage/Storage.java

```diff
@@ -0,0 +1,110 @@
+package muller.storage;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import muller.command.MullerException;
+import muller.task.Task;
+import muller.task.TaskList;
+
+/**
+ * Handles the loading and saving of tasks to and from a file.
+ */
+public class Storage {
+    private String filePath;
+    /**
+     * Constructs a Storage object with the specified file path.
+     *
+     * @param filePath The path to the file where tasks are stored.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+    /**
+     * Loads tasks from the specified file.
+     *
+     * @return An ArrayList of Task objects loaded from the file.
+     * @throws MullerException If there is an issue loading the tasks.
+     */
+    public ArrayList<Task> load() throws MullerException {
```

Comment:
Perhaps a more specific method name such as loadTasks and saveTasks will be better?